policybot/pkg/storage/spanner/read.go

```diff
@@ -239,3 +241,41 @@ func (s *store) ReadBotActivityByID(orgID string, repoID string) (*storage.BotAc
 
 	return &result, nil
 }
+
+func (s *store) ReadTestFlakyIssues(inactiveDays, createdDays int) ([]*storage.Issue, error) {
+	sql := `SELECT * from Issues
+	WHERE TIMESTAMP_DIFF(CURRENT_TIMESTAMP(), UpdatedAt, DAY) > @inactiveDays AND 
+				TIMESTAMP_DIFF(CURRENT_TIMESTAMP(), CreatedAt, DAY) < @createdDays AND
+				State = 'open' AND
+				( REGEXP_CONTAINS(title, 'flake') OR 
+					REGEXP_CONTAINS(body, 'flake') );`
+	stmt := spanner.NewStatement(sql)
+	stmt.Params["inactiveDays"] = inactiveDays
+	stmt.Params["createdDays"] = createdDays
+	var issues []*storage.Issue
+	getIssue := func(row *spanner.Row) error {
+		issue := storage.Issue{}
+		if err := row.ToStruct(&issue); err != nil {
+			return err
+		}
+		issues = append(issues, &issue)
+		return nil
+	}
+	iter := s.client.Single().Query(s.ctx, stmt)
+	defer iter.Stop()
```

Comment:
Any reason not to use the simpler iter.Do?

Reply:
changed.